src/current/v23.2/sso-db-console.md

```diff
@@ -104,7 +115,7 @@ These steps demonstrate how to enable SSO authentication for the DB Console on a
 
```

Comment:
more signposting needed about where this comes from, this is tricky! it's the db console URL with admin- in it